runtime/src/account_info.rs

```diff
@@ -11,29 +15,37 @@ pub struct AccountInfo {
     /// needed to track shrink candidacy in bytes. Used to update the number
     /// of alive bytes in an AppendVec as newer slots purge outdated entries
     stored_size: usize,
```

Comment:
May be worth adding a comment in here that the msb is used for setting zero-lamports.